tests/src/whisk/core/controller/test/EntitlementProviderTests.scala

```diff
@@ -42,7 +44,9 @@ import whisk.core.entity._
  * "using Specs2RouteTest DSL to chain HTTP requests for unit testing, as in ~>"
  */
 @RunWith(classOf[JUnitRunner])
-class EntitlementProviderTests extends ControllerTestCommon {
+class EntitlementProviderTests
+    extends ControllerTestCommon
+    with ScalaFutures {
```

Comment:
futureValue is for the successful case - some of these tests check that the future failed. this would require using "an exception thrown" matcher; i found the eitherValue to be more convenient for specifying the collection of vectors per test.